pom.xml

```diff
@@ -120,6 +120,12 @@
             <version>2.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
```

Comment:
This is only for unit testing, though (Making sure that the largest eigenvalues computed correspond to the largest eigenvalues found by other techniques). It's not required in the main sources.